gamebook/terminal_input.py

```diff
@@ -9,10 +10,11 @@ def input(self, prompt):
         return data
 
     def ask_for_user_inputs(self, options):
-        user_input_string = ""
+        user_input_string = "\n"
         for option in options:
             user_input_string += (f"to {option.show_title()} enter one of the"
                                   f" following {option.show_user_inputs()}\n")
```

Comment:
i didn't move this one because it depends on `option`

Reply:
Maybe move it to there as a template string (with the `f"..."`)

Reply:
you mean just save it there as a template and not import it?
how will it work?

Reply:
import the constant and then use it like you use here:
consts.py:
`USER_INPUT_STRING = "to {title} enter one..."`

terminal_input.py:
`user_input_string = consts.USER_INPUT_STRING.format(title=option.show_title())`

Something like that.

Reply:
oh, got it :+1: that is pretty cool